keyboards/melgeek/z70ultra/z70ultra.c

```diff
@@ -0,0 +1,158 @@
+#include "z70ultra.h"
+#include "config.h"
```

Comment:
```suggestion
```

Reply:
This is not needed, at all, and should be removed.  
```suggestion
```

Specifically, `config.h` gets included in every file by the make system. So this is emphatically not needed.